### PR TITLE
Fix #19: UI crash when chain is not synchronized before NeoCoin claim

### LINKED ISSUE #19: NeoGas Claim unhandled exception
Build 2.0.6411.26926
Advanced > NeoGas Claim
Blockchain not fully sync yet.

image

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ArgumentException: Value does not fall within the expected range.
   at Neo.Core.Blockchain.CalculateBonus(IEnumerable`1 inputs, UInt32 height_end)
   at Neo.UI.ClaimForm.CalculateBonusUnavailable(UInt32 height) in C:\Users\erik\Documents\Visual Studio 2017\Projects\neo-gui\neo-gui\UI\ClaimForm.cs:line 17
   at Neo.UI.ClaimForm.ClaimForm_Load(Object sender, EventArgs e) in C:\Users\erik\Documents\Visual Studio 2017\Projects\neo-gui\neo-gui\UI\ClaimForm.cs:line 25
   at System.Windows.Forms.Form.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2101.1 built by: NET47REL1LAST
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
neo-gui
    Assembly Version: 2.0.6411.26926
    Win32 Version: 2.0.6411.26926
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/neo-gui.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2094.0 built by: NET47REL1LAST
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2093.0 built by: NET47REL1LAST
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Xml.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2099.0 built by: NET47REL1LAST
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Neo
    Assembly Version: 2.0.1.0
    Win32 Version: 2.0.1.0
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Neo.DLL
----------------------------------------
Neo.VM
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.0.0
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Neo.VM.DLL
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
Microsoft.Extensions.Configuration
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Configuration.DLL
----------------------------------------
System.Runtime
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Runtime.dll
----------------------------------------
Microsoft.Extensions.Configuration.Abstractions
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Configuration.Abstractions.DLL
----------------------------------------
System.Collections
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Collections/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Collections.dll
----------------------------------------
Microsoft.Extensions.Configuration.Json
    Assembly Version: 1.0.2.0
    Win32 Version: 1.0.2.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Configuration.Json.DLL
----------------------------------------
Microsoft.Extensions.Configuration.FileExtensions
    Assembly Version: 1.0.2.0
    Win32 Version: 1.0.2.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Configuration.FileExtensions.DLL
----------------------------------------
Microsoft.Extensions.FileProviders.Abstractions
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.FileProviders.Abstractions.DLL
----------------------------------------
Microsoft.Extensions.FileProviders.Physical
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.FileProviders.Physical.DLL
----------------------------------------
Microsoft.Extensions.Primitives
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Primitives.DLL
----------------------------------------
System.Threading.Tasks
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Threading.Tasks/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Threading.Tasks.dll
----------------------------------------
Microsoft.Extensions.FileSystemGlobbing
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.FileSystemGlobbing.DLL
----------------------------------------
System.Runtime.Extensions
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Extensions/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Runtime.Extensions.dll
----------------------------------------
System.IO
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.IO.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 9.0.0.0
    Win32 Version: 9.0.1.19813
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Newtonsoft.Json.DLL
----------------------------------------
System.Numerics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
System.Threading
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Threading/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Threading.dll
----------------------------------------
System.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Linq/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Linq.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
System.IO.Compression
    Assembly Version: 4.1.2.0
    Win32 Version: 4.6.24705.01
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.IO.Compression.DLL
----------------------------------------
Microsoft.AspNetCore.Hosting
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Hosting.DLL
----------------------------------------
Microsoft.AspNetCore.Hosting.Abstractions
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Hosting.Abstractions.DLL
----------------------------------------
Microsoft.Extensions.Logging.Abstractions
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Logging.Abstractions.DLL
----------------------------------------
Microsoft.Extensions.DependencyInjection.Abstractions
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.DependencyInjection.Abstractions.DLL
----------------------------------------
Microsoft.AspNetCore.Server.Kestrel
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Server.Kestrel.DLL
----------------------------------------
Microsoft.AspNetCore.Http.Abstractions
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Http.Abstractions.DLL
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
Microsoft.Extensions.Configuration.EnvironmentVariables
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Configuration.EnvironmentVariables.DLL
----------------------------------------
Microsoft.Extensions.DependencyInjection
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.DependencyInjection.DLL
----------------------------------------
System.ComponentModel
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.ComponentModel/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.ComponentModel.dll
----------------------------------------
Microsoft.AspNetCore.Http.Features
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Http.Features.DLL
----------------------------------------
System.Diagnostics.DiagnosticSource
    Assembly Version: 4.0.1.0
    Win32 Version: 4.6.24705.01
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.Diagnostics.DiagnosticSource.DLL
----------------------------------------
Microsoft.Extensions.PlatformAbstractions
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.PlatformAbstractions.DLL
----------------------------------------
Microsoft.Extensions.Logging
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Logging.DLL
----------------------------------------
Microsoft.AspNetCore.Http
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Http.DLL
----------------------------------------
Microsoft.Extensions.Options
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Options.DLL
----------------------------------------
Microsoft.Extensions.ObjectPool
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.21115
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.ObjectPool.DLL
----------------------------------------
System.Diagnostics.Tracing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Diagnostics.Tracing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Diagnostics.Tracing.dll
----------------------------------------
Microsoft.AspNetCore.Hosting.Server.Abstractions
    Assembly Version: 1.1.1.0
    Win32 Version: 1.1.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.Hosting.Server.Abstractions.DLL
----------------------------------------
System.Collections.Concurrent
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Collections.Concurrent/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Collections.Concurrent.dll
----------------------------------------
System.Reflection
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Reflection/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Reflection.dll
----------------------------------------
System.Runtime.InteropServices.RuntimeInformation
    Assembly Version: 4.0.1.0
    Win32 Version: 4.6.24705.01
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.Runtime.InteropServices.RuntimeInformation.DLL
----------------------------------------
Microsoft.AspNetCore.WebSockets
    Assembly Version: 1.0.1.0
    Win32 Version: 1.0.1.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.AspNetCore.WebSockets.DLL
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 4.1.2.0
    Win32 Version: 4.6.24705.01
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.Numerics.Vectors.DLL
----------------------------------------
System.Globalization
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Globalization/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Globalization.dll
----------------------------------------
System.Net.Sockets
    Assembly Version: 4.1.1.0
    Win32 Version: 4.6.24705.01
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.Net.Sockets.DLL
----------------------------------------
Microsoft.EntityFrameworkCore
    Assembly Version: 1.0.3.0
    Win32 Version: 1.0.3.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.EntityFrameworkCore.DLL
----------------------------------------
System.Interactive.Async
    Assembly Version: 3.0.0.0
    Win32 Version: 3.0.0.0
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.Interactive.Async.DLL
----------------------------------------
Microsoft.Extensions.Caching.Memory
    Assembly Version: 1.0.2.0
    Win32 Version: 1.0.2.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Caching.Memory.DLL
----------------------------------------
Remotion.Linq
    Assembly Version: 2.1.0.0
    Win32 Version: 2.1.1.30000
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Remotion.Linq.DLL
----------------------------------------
Microsoft.Extensions.Caching.Abstractions
    Assembly Version: 1.0.2.0
    Win32 Version: 1.0.2.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Extensions.Caching.Abstractions.DLL
----------------------------------------
Microsoft.Data.Sqlite
    Assembly Version: 1.0.1.0
    Win32 Version: 1.0.1.21122
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.Data.Sqlite.DLL
----------------------------------------
Microsoft.EntityFrameworkCore.Sqlite
    Assembly Version: 1.0.3.0
    Win32 Version: 1.0.3.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.EntityFrameworkCore.Sqlite.DLL
----------------------------------------
Microsoft.EntityFrameworkCore.Relational
    Assembly Version: 1.0.3.0
    Win32 Version: 1.0.3.30217
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/Microsoft.EntityFrameworkCore.Relational.DLL
----------------------------------------
System.Linq.Expressions
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Linq.Expressions/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Linq.Expressions.dll
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 4.7.2101.1 built by: NET47REL1LAST
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/mscorlib/v4.0_4.0.0.0__b77a5c561934e089/mscorlib.dll
----------------------------------------
System.ComponentModel.DataAnnotations
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.ComponentModel.DataAnnotations/v4.0_4.0.0.0__31bf3856ad364e35/System.ComponentModel.DataAnnotations.dll
----------------------------------------
System.Collections.Immutable
    Assembly Version: 1.2.1.0
    Win32 Version: 4.6.24705.01
    CodeBase: file:///C:/AntShares/neo-gui-windows/Neo/System.Collections.Immutable.DLL
----------------------------------------
System.Transactions
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Transactions/v4.0_4.0.0.0__b77a5c561934e089/System.Transactions.dll
----------------------------------------
System.EnterpriseServices
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.EnterpriseServices/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.EnterpriseServices.dll
----------------------------------------
Microsoft.CSharp
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Microsoft.CSharp/v4.0_4.0.0.0__b03f5f7f11d50a3a/Microsoft.CSharp.dll
----------------------------------------
System.Dynamic
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Dynamic/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Dynamic.dll
----------------------------------------
System.Security
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2046.0 built by: NET47REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Security/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Security.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

### PR DESCRIPTION
**Pull request by danielcrenna**
Filter out any coin references we can't find in local chain, since CalculateBonus is more strict and assumes a complete height.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Project Setup
+=============
+
+To build and run locally, you need to clone and build https://github.com/neo-project/leveldb first, 
+then copy `libleveldb.dll` to the working directory (i.e. /bin/Debug, /bin/Release)
```
Comment (erikzhang) [inline README.md]: Only need to build leveldb for Windows.
For linux, you can install it easily:
```
yum install leveldb-devel
```

File: neo-gui.sln [hunk 2]
```diff
@@ -1,10 +1,15 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26430.15
+VisualStudioVersion = 15.0.26730.3
```
Comment (erikzhang) [inline neo-gui.sln]: Please don't change .sln file.

**Review by erikzhang — CHANGES_REQUESTED** [review]

File: neo-gui.sln (hunk 2 above)
Comment (danielcrenna) [reply neo-gui.sln]: sorry, reverted

File: README.md (hunk 1 above)
Comment (danielcrenna) [reply README.md]: added to README

**Review by erikzhang — APPROVED** [review]

**Pull request merged**